Swiftfin tvOS/Views/SelectUserView/SelectUserView.swift

```diff
@@ -278,26 +292,32 @@ struct SelectUserView: View {
         }
     }
 
-    // MARK: - Body
+    // MARK: - Functions
 
-    var body: some View {
-        ZStack {
-            if viewModel.servers.isEmpty {
-                emptyView
+    private func didDelete(_ server: ServerState) {
+        viewModel.send(.getServers)
+
+        if case let SelectUserServerSelection.server(id: id) = serverSelection, server.id == id {
+            if viewModel.servers.keys.count == 1, let first = viewModel.servers.keys.first {
+                serverSelection = .server(id: first.id)
             } else {
-                userView
+                serverSelection = .all
             }
         }
-        .ignoresSafeArea()
-        .navigationBarBranding()
-        .onAppear {
-            viewModel.send(.getServers)
 
-            splashScreenImageSource = makeSplashScreenImageSource(
-                serverSelection: serverSelection,
-                allServersSelection: .all
-            )
+        // change splash screen selection if necessary
+//            selectUserAllServersSplashscreen = serverSelection
+    }
+
+    private func didAppear() {
+        viewModel.send(.getServers)
 
+        splashScreenImageSource = makeSplashScreenImageSource(
+            serverSelection: serverSelection,
+            allServersSelection: .all
+        )
+
+        // TODO: Is this necessary anymore?
```

Comment:
No, this isn't necessary anymore. This was just from my original implementation for testing the views.